package.json

```diff
@@ -1,115 +1,125 @@
 {
-  "name": "streamline",
-  "version": "0.0.0",
-  "license": "MIT",
+  "name": "@getluko/streamline",
+  "version": "0.0.53",
```

Comment:
0.1.0

Reply:
I will do it when the github action for publishing to npm is ready